lib/rouge/lexers/gdscript.rb

```diff
@@ -0,0 +1,201 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    class GDScript < RegexLexer
+      title "GDScript"
+      desc "The Godot Engine programming language (https://godotengine.org/)"
+      tag 'gdscript'
+      aliases 'gd', 'gdscript'
+      filenames '*.gd'
+      mimetypes 'text/x-gdscript', 'application/x-gdscript'
+
+      affix_long  = '[rR]|[uUbB][rR]|[rR][uUbB]|[@]'
```

Comment:
Thanks for fixing this. I have no idea why I suggested the method be called `class.affix_long` rather than `self.affix_long`. I hope that didn't put you on the wrong track!